pkg/agent/flowexporter/connections/conntrack_ovs.go

```diff
@@ -109,28 +111,29 @@ func flowStringToAntreaConnection(flow string, zoneFilter uint16) (*flowexporter
 		if strings.Contains(fs, "reply") {
 			isReply = true
 		}
-		if !strings.Contains(fs, "=") {
+		switch {
+		case !strings.Contains(fs, "="):
```

Comment:
Could we make this little more clear by checking protocol strings (keys) from global "protocols" map directly?

Reply:
Sure. Done.